kola/tests/docker/docker.go

```diff
@@ -610,3 +610,21 @@ func testDockerInfo(expectedFs string, c cluster.TestCluster) {
 		c.Errorf("runc was not in runtimes: %+v", info.Runtimes)
 	}
 }
+
+// hasSecurityOptions strictly checks that at least one of
+// the docker security option is enabled (seccomp, selinux)
+func hasSecurityOptions(opts []string) bool {
+	var selinux, seccomp bool
```

Comment:
Wouldn't just `len(opts) > 0` work?

Reply:
Good point ! Previous test was strictly checking for `seccomp` and `selinux` in the security options.

So
```go
len(opts) > 0
```

would work but we won't strictly test the enabled security options. (could be `apparmor`, `toto` or whatever). I tried to stay closer from the current test and to just allow some releases to only have `seccomp` as security options.